docs/architecture/smart-rollups.md

```diff
@@ -91,7 +91,8 @@ Eventually include:
 
 The general flow of a Smart Rollup goes through these phases:
 
-1. Origination: A user called the _rollup operator_ originates the Smart Rollup to layer 1 and one or more users start nodes based on that Smart Rollup to independently verify its operation.
+1. Origination: A user originates the Smart Rollup to layer 1.
```

Comment:
The origination is also referred to as the "genesis" of the rollup (genesis level = level of L1 block in which origination operation was included). Not sure this is worth mentioning in this user centric documentation but if it pops up elsewhere maybe you can link back to here.

Reply:
Good tip. The Origination section mentions the genesis commitment -- maybe that's enough for now.

Reply:
Sounds good :)